go/filemanager/pfsmodules/upload.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 // Config is global config object for pfs commandline
-var Config = config.ParseDefaultConfig()
+var Config *config.SubmitConfig
```

Comment:
Why `Config` global is defined in `upload.go`, should it be placed in a common file?

Reply:
Done